readthedocs/templates/core/project_detail_right.html

```diff
@@ -91,6 +91,15 @@ <h3>{% trans "Project Privacy Level" %}</h3>
   </div>
 {% endblock %}
 
+{% block project-default-branch %}
+  <h3>{% trans "Project Default Branch" %}{# trans does not work here. #}</h3>
```

Comment:
Why that comment `{# trans does not work here. #}`?

Reply:
`{% trans "Project Default Branch" %}`
The language for this should change when we change the website language. But somehow it doesn't know the right word for 'Default' in other languages and therefore it stays in English language only. I couldn't figure out why that happens. That's why I wrote the comment.

Reply:
that's because of missing translations for that word, RTD uses transifex for collaborating on i18n https://docs.readthedocs.io/en/latest/i18n.html. It's safe to remove that comment.

Reply:
Oh. Okay! I will remove the comment.